### PR TITLE
[FR] Update docstrings

### LINKED ISSUE #17021: Improve docstrings for TA and FR
Not only explaining what the actual purpose of the property/class is, but also following: https://review.docs.microsoft.com/en-us/help/contribute-ref/contribute-ref-how-to-write-.net-docs?branch=master

### PR DESCRIPTION
**Pull request by maririos**
- Updates docstrings
- Remove locale from links

Fixes: https://github.com/Azure/azure-sdk-for-net/issues/17021

### CODE REVIEW

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormPage.cs [hunk 1]
```diff
@@ -41,7 +42,7 @@ internal FormPage(PageResult pageResult, IReadOnlyList<ReadResult> readResults,
         /// <param name="width">The width of the image/PDF in pixels/inches, respectively.</param>
         /// <param name="height">The height of the image/PDF in pixels/inches, respectively.</param>
         /// <param name="textAngle">The general orientation of the text in clockwise direction, measured in degrees between (-180, 180].</param>
-        /// <param name="unit">The unit used by the width, height and <see cref="FieldBoundingBox"/> properties. For images, the unit is &quot;pixel&quot;. For PDF, the unit is &quot;inch&quot;.</param>
+        /// <param name="unit">The unit used by the width, height and <see cref="FieldBoundingBox"/> properties. For images, the unit is "pixel". For PDF, the unit is "inch".</param>
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormPage.cs:45]: Not gonna lie to you, I always saw `&quot;` being used in FR docstrings, so I assumed that directly using quotes wouldn't work (in code or in the generated docs). Did it change recently? If not, any idea as to why we did it like this?

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormSelectionMark.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ internal FormSelectionMark(FieldBoundingBox boundingBox, int pageNumber, string
         public float Confidence { get; }
 
         /// <summary>
-        /// Selection mark state value.
+        /// Selection mark state value, like Selected or Unselected.
         /// </summary>
         public SelectionMarkState State { get; }
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormSelectionMark.cs:33]: `like Selected or Unselected` makes it sound like there are other possible states (are there?). It may be worth rephrasing it a bit to make it clear that these are the only states.

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs [hunk 3]
```diff
@@ -6,13 +6,17 @@
 namespace Azure.AI.FormRecognizer
 {
     /// <summary>
-    /// The set of options that can be specified when calling a recognize content method
-    /// to configure the behavior of the request.
+    /// The set of options that can be specified when calling a Recognize Content method
+    /// to configure the behavior of the request. For example, specify the content type of the
+    /// form, the language of the form, and the custom page numbers in a multi-page document to analyze.
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs:11]: `custom page numbers` feels a bit weird to me. I think I'm reading it more like `custom (page numbers)` than `(custom page) numbers`. Maybe something like:
```
...the language of the form, and the page numbers in a multi-page document to analyze.
...the language of the form, and the set of pages in a multi-page document to analyze.
...the language of the form, and the input page numbers in a multi-page document to analyze.
...the language of the form, and which pages in a multi-page document to analyze.
```

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs [hunk 4]
```diff
@@ -6,13 +6,17 @@
...
     /// </summary>
     public class RecognizeContentOptions
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="RecognizeContentOptions"/> class.
+        /// Initializes a new instance of the <see cref="RecognizeContentOptions"/> class which
+        /// allows to set options that can be specified when calling a Recognize Content method
+        /// to configure the behavior of the request. For example, specify the content type of the
+        /// form, the language of the form, and the custom page numbers in a multi-page document to analyze.
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs:19]: If you decide to change the class description, just a reminder to update this one as well.

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs [hunk 5]
```diff
@@ -14,7 +14,7 @@ namespace Azure.AI.FormRecognizer.Training
 {
     /// <summary>
     /// The client to use to connect with the Form Recognizer Azure Cognitive Service to train models from
-    /// custom forms.  It also supports listing and deleting trained models, as well as accessing account
+    /// custom forms. It also supports listing, copy, and deleting trained models, create composed models, and accessing account
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs:17]: I'd rephrase it as:
```
It also supports listing, *copying*, and deleting trained models, *creating* composed models, and accessing...
```

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs [hunk 6]
```diff
@@ -14,7 +14,7 @@ namespace Azure.AI.FormRecognizer
     /// <summary>
     /// The client to use to connect to the Form Recognizer Azure Cognitive Service to recognize
     /// information from forms and images and extract it into structured data. It provides the ability to analyze receipts,
-    /// to recognize form content, and to extract fields from custom forms with models trained on custom form types.
+    /// business cards, invoices, to recognize form content, and to extract fields from custom forms with models trained on custom form types.
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:17]: Maybe:
```
It provides the ability to analyze receipts, business cards, *and* invoices, to recognize form content...
```

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs [hunk 7]
```diff
@@ -120,7 +120,8 @@ protected FormRecognizerClient()
         /// Recognizes layout elements from one or more passed-in forms.
         /// </summary>
         /// <param name="form">The stream containing one or more forms to recognize elements from.</param>
-        /// <param name="recognizeContentOptions">A set of options available for configuring the recognize request.</param>
+        /// <param name="recognizeContentOptions">A set of options available for configuring the recognize request. For example, specify the content type of the
+        /// form, the language of the form, and the custom page numbers in a multi-page document to analyze.</param>
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:124]: Same comment here about `custom page numbers`.

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs [hunk 8]
```diff
@@ -158,7 +159,8 @@ public virtual RecognizeContentOperation StartRecognizeContent(Stream form, Reco
...
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:163]: Same comment here about `custom page numbers`.

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs [hunk 9]
```diff
@@ -196,7 +198,8 @@ public virtual async Task<RecognizeContentOperation> StartRecognizeContentAsync(
         /// Recognizes layout elements from one or more passed-in forms.
         /// </summary>
         /// <param name="formUri">The absolute URI of the remote file to recognize elements from.</param>
-        /// <param name="recognizeContentOptions">A set of options available for configuring the recognize request.</param>
+        /// <param name="recognizeContentOptions">A set of options available for configuring the recognize request. For example, specify the content type of the
+        /// form, the language of the form, and the custom page numbers in a multi-page document to analyze.</param>
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:202]: Same comment here about `custom page numbers`.

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs [hunk 10]
```diff
@@ -232,7 +235,8 @@ public virtual RecognizeContentOperation StartRecognizeContentFromUri(Uri formUr
...
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:239]: Same comment here about `custom page numbers`.

**Review by kinelski — APPROVED** [review]
LGTM. Left some comments, but none is blocking.

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValueType.cs [hunk 11]
```diff
@@ -58,7 +58,7 @@ public enum FieldValueType
         Dictionary,
 
         /// <summary>
-        /// Used for <see cref="FieldValueSelectionMark"/> type.
+        /// Used for Selection Mark type.
```
[the remark this answers is not in the document]
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValueType.cs:61]: This is a tricky one. So `FieldValueSelectionMark` was wrong because that is an internal type. It should be `SelectionMarkState` but saying `Used for SelectionMarkState type` sounded weird for me... so I just said Selection Mark.
I will add `SelectionMarkState` as it is the real type

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/AccountProperties.cs [hunk 12]
```diff
@@ -29,7 +29,7 @@ internal AccountProperties(int customModelCount, int customModelLimit)
         }
 
         /// <summary>
-        /// The current count of trained custom models.
+        /// The current count of trained custom models in this subscription.
```
Comment (kristapratico) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/AccountProperties.cs:32]: should it say in this subscription here? Or is this per account?

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormPage.cs (hunk 1 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormPage.cs:45]: It is displayed properly. I just find it hard to read. 
To avoid confusion I am just going to remove it all together

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormSelectionMark.cs (hunk 2 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormSelectionMark.cs:33]: For now there aren't, but in the future `Undefined` might be included

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs (hunk 5 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs:17]: ugh English!
Thank you!!

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs [hunk 13]
```diff
@@ -273,7 +277,8 @@ public virtual async Task<RecognizeContentOperation> StartRecognizeContentFromUr
         /// <para>See <a href="https://aka.ms/formrecognizer/receiptfields"/> for a list of available fields on a receipt.</para>
         /// </summary>
         /// <param name="receipt">The stream containing the one or more receipts to recognize values from.</param>
-        /// <param name="recognizeReceiptsOptions">A set of options available for configuring the recognize request.</param>
+        /// <param name="recognizeReceiptsOptions">A set of options available for configuring the recognize request. For example, specify the content type of the
+        /// form, the locale of the form, or whether or not to include form elements.</param>
```
Comment (kristapratico) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:281]: since we're listing all the options here, it might be worth mentioning that the includeFieldElements option also controls whether you'll get back all the lines on the page. Or maybe on the parameter itself, actually.

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValueType.cs (hunk 11 above)
[the remark this answers is not in the document]
Comment (kinelski) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValueType.cs:61]: I see. No major concerns from my side, so your call :)

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/AccountProperties.cs (hunk 12 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/AccountProperties.cs:32]: ohh you right. account is more appropriate. I will fix

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs (hunk 13 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:281]: So the [parameter](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeReceiptsOptions.cs#L22) itself says:
>Whether or not to include form elements such as lines and words in addition to form fields.

Do you think I need to be more specific on it?

Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:281]: I saw Python's and I like it more :P I will copy that :)
>Whether or not to include all lines per page and field elements such as lines, words,
            and selection marks for each form field.

Comment (kristapratico) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:281]: Hmm not sure I would connect that with the list of `lines` being populated on `FormPage`. I recently changed my doc for this to "Whether or not to include all lines per page and field elements such as lines, words, and selection marks for each form field.".

Comment (kristapratico) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:281]: 😄 I didn't see your previous post until after I commented

Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs:281]: No worries :)
